Tools/px_uploader.entrypoint

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -eu
+
+/bin/px_uploader.py --udp-addr=192.168.200.101 --udp-port=14541 --port=/dev/px4serial --baud-flightstack=57600,115200,1000000,2000000 --baud-bootloader=2000000 --use-protocol-splitter-format px4_fmu-v5_ssrc-*.px4 px4_fmu-v5x_ssrc-*.px4 ssrc_saluki-v1_default-*.px4
+/bin/px4-post-update-init -a :14540 -m outdoor
```

Comment:
Code-wise this PR now looks ready, thanks! :clap: 

Configuration-wise, I had one outstanding question from earlier comment:

> Now thinking more of this, does this default mode break our most common case which is indoor flight?

So this line sets outdoor config, which breaks our most common use case, which is indoor flight?

Should we take this as parameter to the entrypoint, so we give the flight mode from manifest?

Then we could change the manifest entrypoint options to have the flight mode as dynamic parameter which fog_hyper is responsible for giving? That way we can eventually implement changing the flight mode from the web UI and have it propagated to here?

```diff
@@ -85,7 +85,7 @@
        {
            "Name": "px4-firmware",
            "Repository": "https://github.com/tiiuae/px4-firmware",
            "Revision": "81a93c4e636930433c5116339679cd104163d491",
            "Artifacts": [
                {
                    "Digest": "3f0323bf26138c856fb2e685b451432bdff9ee9172fe68e49181d56574473f4d",
                    "ImageRef": "ghcr.io/tiiuae/px4-firmware:sha-81a93c4"
                }
            ],
            "Notes": "(Pre-loaded container image.) Flashes PX4 firmware. Shell entrypoint override to get wildcard expansion.",
            "ContainerOptions": {
                "RunsOnVM": "$hypervisor.one-off-util",
                "DockerOpts": [
                     "--device=/dev/px4serial:/dev/px4serial"
                 ],
                 "EntrypointOpts": [
-                    "--flightmode=indoor_lighthouse"
+                    "--flightmode=${drone.FLIGHT_MODE}"
                 ]
             }
         },
```

Reply:
That sounds good. Let's make it that way